ppr-ui/src/components/mhrRegistration/HomeOwners/HomeOwnersTable.vue

```diff
@@ -446,6 +468,15 @@ export default defineComponent({
       }),
       isUngroupedTenancy: computed((): boolean => {
         return [HomeTenancyTypes.SOLE, HomeTenancyTypes.JOINT].includes(getHomeTenancyType())
+      }),
+      disableDelete: computed(() => {
+      }),
+      transWillSupportDoc: null as SupportingDocumentsOptions,
```

Comment:
Super minor but unused computed value here and maybe keep the `transWillSupportDoc` above the computed values for consistency.

Reply:
Good point, will move move it.